### PR TITLE
Feature/Motorhead Memory

### PR DESCRIPTION
**Pull request by ivalkshfoeif**
## Issue

#368

# Note

https://github.com/FlowiseAI/Flowise/issues/368#issuecomment-1623036178

### CODE REVIEW

**Comment by HenryHengZJ** [comment]
Tested locally works fine for me:
image

and when retrieve the message:
image

File: packages/components/nodes/memory/MotorheadMemory/MotorheadMemory.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                description: 'if empty, chatId will be used automatically',
+                default: '',
+                additionalParams: true,
+                optional: true
+            },
+            {
+                label: 'API Key',
+                name: 'apiKey',
+                type: 'string',
+                optional: true
```
Comment (HenryHengZJ) [inline packages/components/nodes/memory/MotorheadMemory/MotorheadMemory.ts:51]: can we add `additionalParams: true` and `description: 'Only needed when using hosted solution https://getmetal.io'`

**Comment by HenryHengZJ** [comment]
run `yarn lint-fix` to fix the linting issues

**Comment by ivalkshfoeif** [comment]
Node screenshot:
Screen Shot 2023-07-10 at 12 36 09 AM
Screen Shot 2023-07-10 at 12 35 46 AM

**Review by HenryHengZJ — APPROVED** [review]
looks good! thanks @ivalkshfoeif

**Pull request merged**